front/lib/api/assistant/global_agents.ts

```diff
@@ -423,34 +423,25 @@ export async function getGlobalAgents(
 
   // For now we retrieve them all
   // We will store them in the database later to allow admin enable them or not
-  const globalAgents = [
-    await _getHelperGlobalAgent(user),
-    await _getGPT4GlobalAgent(),
-    await _getGPT35TurboGlobalAgent(),
-    await _getClaudeGlobalAgent(),
-    await _getClaudeInstantGlobalAgent(),
-  ];
-
-  const slackAgent = await _getSlackGlobalAgent(auth);
-  if (slackAgent) {
-    globalAgents.push(slackAgent);
-  }
-
-  const googleDriveAgent = await _getGoogleDriveGlobalAgent(auth);
-  if (googleDriveAgent) {
-    globalAgents.push(googleDriveAgent);
-  }
-  const notionAgent = await _getNotionGlobalAgent(auth);
-  if (notionAgent) {
-    globalAgents.push(notionAgent);
-  }
-  const githubAgent = await _getGithubGlobalAgent(auth);
-  if (githubAgent) {
-    globalAgents.push(githubAgent);
-  }
-  const dustAgent = await _getDustGlobalAgent(auth);
-  if (dustAgent) {
-    globalAgents.push(dustAgent);
+  const agentCandidates = await Promise.all([
```

Comment:
I would do it with
```
await Promise.all(GLOBAL_AGENTS_SID.values().map((a) => getGlobalAgent(auth, a.sId)))
```
The reason is, if we add or remove a global agent, it's easy to forget to update the list here 
Better to have one single source of truth for the global agent list

As a nice to have it is more concise

Reply:
Now that ordering is supposed to be handled client-side I guess it's a fair ask?

Reply:
@philipperolet agree now. Thanks for catching it again!